src/httpServer/middleware.ts

```diff
@@ -112,6 +114,23 @@ export function KoaMiddleware(...koaMiddleware: Koa.Middleware[]) {
   return clsdec;
 }
 
+/**
+ * Define Koa middleware that is applied in order to each endpoint in this class.
```

Comment:
To each endpoint in this class or every class?

Reply:
Woops.